modules/components/src/Aggs/DatesAgg.css

```diff
@@ -0,0 +1,68 @@
+.dateInput {
```

Comment:
Could you use emotion for the styles? or prefix them, these class names are just a little too generic.

Reply:
opted for CSS so it can be overridden if needed :)